packages/react-ui-kit/webpack.config.js

```diff
@@ -36,6 +36,9 @@ module.exports = {
   resolve: {
     alias: {
       '@wireapp/react-ui-kit': path.resolve(__dirname, 'src'),
+      // Needed until libs have support for emotion v11
+      // https://github.com/infinum/emotion-normalize/pull/12
+      '@emotion/core': '@emotion/react',
```

Comment:
We need to redirect calls from `@emotion/core` to `@emotion/react` until `emotion-normalize` supports it.

Reply:
https://github.com/infinum/emotion-normalize/pull/12